src/search/engines/_sedefaults/init.go

```diff
@@ -9,14 +9,35 @@ import (
 	"github.com/hearchco/hearchco/src/config"
 	"github.com/hearchco/hearchco/src/search/bucket"
 	"github.com/hearchco/hearchco/src/search/engines"
+	"github.com/hearchco/hearchco/src/search/useragent"
 	"github.com/rs/zerolog/log"
 )
 
-// it's okay to return pointers to collectors since colly.NewCollector() returns a pointer
 func InitializeCollectors(ctx context.Context, engineName engines.Name, options engines.Options, settings config.Settings, timings config.CategoryTimings, relay *bucket.Relay) (*colly.Collector, *colly.Collector) {
-	col := colly.NewCollector(colly.MaxDepth(1), colly.UserAgent(options.UserAgent), colly.Async())
-	pagesCol := colly.NewCollector(colly.MaxDepth(1), colly.UserAgent(options.UserAgent), colly.Async())
+	// get random user agent and corresponding Sec-Ch-Ua header
+	userAgent, secChUa := useragent.RandomUserAgentWithHeader()
 
+	// create collectors
+	col := colly.NewCollector(
+		colly.Async(),
+		colly.MaxDepth(1),
+		colly.UserAgent(userAgent),
+		colly.IgnoreRobotsTxt(),
+		colly.Headers(map[string]string{
```

Comment:
Should we set more headers here like with image proxy? PR #318 had some issues so I had to revert it.